packages/genesys-spark-components/src/components/stable/gux-popover-list/gux-popover-list.tsx

```diff
@@ -74,6 +79,40 @@ export class GuxPopoverList {
   @Event()
   guxdismiss: EventEmitter<void>;
 
+  /**
+   * Fired when the popover is open
+   */
+  @Event()
+  guxopen: EventEmitter<void>;
+
+  /**
+   * Fired when the popover is close
+   */
+  @Event()
+  guxclose: EventEmitter<void>;
+
+  @Watch('isOpen')
+  onIsOpenChange(isOpen: boolean): void {
+    if (isOpen) {
+      this.guxopen.emit();
+
+      console.log('Focus', this.root);
+      afterNextRender(() => {
+        this.root.focus();
```

Comment:
I think I would expect the first `gux-list-item` to get focus, not the `gux-list` itself

Reply:
There are other issues with that so I don't think this PR is a good place to address this.